canonical-schema.json

```diff
@@ -5,7 +5,7 @@
    , " It enforces just a general structure for all exercises,    "
    , " without specifying how the test data should be organized   "
    , " for each type of test. We do this to keep generality and   "
-   , " allow support for tests the do not fit well in the         "
+   , " allow support for tests that do not fit well in the         "
```

Comment:
for the purpose of allowing all the right-side quotes to line up, let's delete a space from the right side as well!

Reply:
Done!